docs/orchestrating-elastic-stack-applications/kibana.asciidoc

```diff
@@ -206,6 +206,18 @@ To deploy more than one instance of Kibana, all the instances must share a same
 * `xpack.reporting.encryptionKey`
 * `xpack.encryptedSavedObjects.encryptionKey`
 
+[TIP]
+====
+If you need to access these encryption keys, you can find them using the `kubectl get secrets` command. For example, link:{kibana-ref}/xpack-security-secure-saved-objects.html#encryption-key-rotation[to rotate the encryption keys] extract the current value of `xpack.encryptedSavedObjects.encryptionKey` in order to set it as a decryption-only key under `xpack.encryptedSavedObjects.keyRotation.decryptionOnlyKeys`.
```

Comment:
```suggestion
If you need to access these encryption keys, you can find them using the `kubectl get secrets` command. 
```

Sorry my previous comment was maybe confusing. I meant we should not talk about key rotation here (Or if we really must then pull it out into its own section with more context and full examples)

Reply:
Thanks! I agree 100%.